.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: false
+
+language: java
+
+jdk:
+  - oraclejdk8
+  - oraclejdk7
```

Comment:
Any particular reason for having both 7 and 8? Probably not a bad idea, I just don't know if there is a specific reason to do so.